docs/kafka/nodejs-kafka/README.adoc

```diff
@@ -74,14 +74,18 @@ END GENERATED ATTRIBUTES
 ////
 
 [id="chap-using-nodejs"]
-= Using Node.js applications with Kafka instances in {product-long-kafka}
+= Manually connecting Node.js applications to Kafka instances in {product-long-kafka}
```

Comment:
I wonder if we should use the word  "Manually" (as it might have a "negative" ring to it). We can say that that is implicit and we want to call out when we connect "automatically.

Reply:
@DuncanDoyle Yeah, I did hum and haw about this. The problem (as I see it) is that it's too much of a stretch to say that service contexts make things _automatic_ - just more automated. I've indicated that here in the note on line 87.

Without being able to call one approach manual or the other automatic, it's hard to draw the appropriate distinction to help users pick the right guide from our landing page. That's what these updates are geared towards. I guess I didn't see it as _such_ a bad thing that we use the quick starts to paint contexts as a favorable alternative to manual configuration work. Again, the note on line 87 is aimed at doing that.

WDYT?

Reply:
Yeah, that explanation makes sense.